src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -67,9 +67,9 @@
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.11020001-fabe022e" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.29-11847" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="4.0.3-11847" />
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.3" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.30-dc" />
```

Comment:
These local package changes allow me to build locally and show everything E2E. However, these packages are not yet published, so the build on this branch currently fails. I'm considering publishing a temporary set of packages from my SDK build to get this CI build passing.